ios/PurchasesHybridCommon/PurchasesHybridCommon/RCCommonFunctionality.h

```diff
@@ -40,15 +40,15 @@ __attribute((deprecated("Use the set<NetworkId> functions instead.")));
 
 + (void)logOutWithCompletionBlock:(RCHybridResponseBlock)completion;
 
-+ (void)createAlias:(nullable NSString *)newAppUserId completionBlock:(RCHybridResponseBlock)completion 
-__attribute((deprecated("Use logIn instead.")));
+//+ (void)createAlias:(nullable NSString *)newAppUserId completionBlock:(RCHybridResponseBlock)completion 
+//__attribute((deprecated("Use logIn instead.")));
 
-+ (void)identify:(NSString *)appUserId
- completionBlock:(RCHybridResponseBlock)completion
-__attribute((deprecated("Use logIn instead.")));
+//+ (void)identify:(NSString *)appUserId
+// completionBlock:(RCHybridResponseBlock)completion
+//__attribute((deprecated("Use logIn instead.")));
 
-+ (void)resetWithCompletionBlock:(RCHybridResponseBlock)completion
-__attribute((deprecated("Use logOut instead.")));
+//+ (void)resetWithCompletionBlock:(RCHybridResponseBlock)completion
+//__attribute((deprecated("Use logOut instead.")));
```

Comment:
These can be deleted, right?

Reply:
They're obsoleted now instead of deprecated.

Reply:
yeah, kill them with fire